test/test_rentlinx.rb

```diff
@@ -1,8 +1,23 @@
 require 'minitest/autorun'
 require 'rentlinx'
+require 'vcr'
 
 # Test for Rentlinx module
-class TestRentlinx < MiniTest::Unit::TestCase
+class TestRentlinx < MiniTest::Test
+  def setup
+    # We use hide the real values using VCR
+    @auth_key = ENV['RENTLINX_AUTH_KEY'] || '<AUTH_KEY>'
+    @password = ENV['RENTLINX_PASSWORD'] || '<PASSWORD>'
+    @username = ENV['RENTLINX_USERNAME'] || '<USERNAME>'
+    @site_url = ENV['RENTLINX_SITE_URL'] || 'http://localhost'
```

Comment:
> Could it be moved out to test_helper.rb? Not important right now but it's a fair amount of boilerplate.

Yes, that's the plan. We should have a RentlinxTest parent class that provides this setup.